.github/workflows/lib-build-and-push.yml

```diff
@@ -98,7 +98,7 @@ jobs:
           echo "CIBW_TEST_COMMAND=true" >> $GITHUB_ENV;
           echo "CIBW_TEST_COMMAND_WINDOWS=(exit 0)" >> $GITHUB_ENV;
           echo "CIBW_TEST_SKIP=*" >> $GITHUB_ENV;
-          echo "CIBW_SKIP=cp2* cp36* pp36* cp37* pp37* *i686 *musllinux*" >> $GITHUB_ENV;
+          echo "CIBW_SKIP=cp2* cp36* pp36* cp37* pp37* pp38* *i686 *musllinux*" >> $GITHUB_ENV;
```

Comment:
Why only pp38 is added here and not cp38? I don't see this explained in PR cover letter / commit message, and it is not obvious to me.